bookkeeper-server/conf/bk_server.conf

```diff
@@ -574,3 +574,42 @@ zkEnableSecurity=false
 # This provides the plugin run extra services along with a bookie server.
 #
 # extraServerComponents=
+
+
+#############################################################################
```

Comment:
Can you add those settings to  documentation? https://github.com/apache/bookkeeper/blob/master/site/_data/config/bk_server.yaml

Reply:
Sure, will do that before merging